Opportunities Page Revisions

ezramoss opened the pull request:
This resolves issue #5

Changed font colors, font sizes, font weights, background colors, rounded corners, filter icon, and scrollbar gap to be more inline with Figma design.

ezramoss committed:
Opportunities View Changes

In Opportunities view, changed header, added tabs. Added filter and sort icon buttons. Added search bar. Changed card button text and color. Changed sidebar color of active tab.

ezramoss committed:
Refactored Dashboard Views & Minor CSS Changes

Removed repeat code/CSS, Removed App.tsx/css code, moved some css to index.css. fixed background color display error on opportunities tab.  added scroll bar to project cards. refactored some cards to display via mappings.

ezramoss committed:
Added Inter Font, minor padding changes.

ezramoss committed:
Initial Profile View

Initial Profile View

ezramoss committed:
Updated Profile Page

Refactored .TSX and .CSS to be more readable, edited experience font, added total yoe.

ezramoss committed:
Visual Enhancements

- Edited Profile Page to reduce gap at role listing
- Edited Opportunities page to make card/tabs text and colors more inline with Figma Design

ezramoss committed:
Updated Opportunities Page / Card Component

Changed font colors, font sizes, font weights, background colors, rounded corners, filter icon, and scrollbar gap to be more inline with Figma design.

ezramoss commented:
@AfnanZ Can you please review my changes and let me know your thoughts? Thank you.

AfnanZ commented:
Hey @ezramoss, thanks for working on the opportunity page. It resembles the Figma design quite closely. A couple of things to tweak. 
The scroll bar is not aligned with the card, it's a bit off. Can we also shorten the bar to 97px perhaps?
<img width="424" height="585" alt="Screenshot 2025-07-16 at 2 33 06 PM" src="https://github.com/user-attachments/assets/d6d9d0d1-261b-4ef7-9dae-c4f2ed1fa875" />

AfnanZ commented:
@ezramoss Have you implemented the design for the side nav bar, or am I still seeing the old version?

<img width="240" height="728" alt="Screenshot 2025-07-16 at 2 58 47 PM" src="https://github.com/user-attachments/assets/4ca803e4-add4-4a4a-9b46-f5cd99ee9aff" />

ezramoss committed:
Sidebar fixes

Updated sidebar with new Figma design. Updated scroll bar on opportunities page. Created basic settings and tools page.
